src/server/scripts/Spells/spell_script_loader.cpp

```diff
@@ -1,52 +1,54 @@
-/*
```

Comment:
ooks like you pushed the file with CR+LF line endings instead of expected LF so it shows up as changing the entire file - this needs to be fixed